docs/tutorial/tut_xts.md

```diff
@@ -29,29 +29,36 @@ which is the code for `stg_customer`)
 
 Create a new dbt model as before. We'll call this one `xts_customer.sql`. 
 
-`xts_customer.sql`
-```jinja
-{{ dbtvault.xts(src_pk=src_pk, src_satellite=src_satellite, src_ldts=src_ldts,
-                src_source=src_source, source_model=source_model)              }}
-```
+=== "xts_customer.sql"
+
+    ```jinja
+    {{ dbtvault.xts(src_pk=src_pk, src_satellite=src_satellite, src_ldts=src_ldts,
+                    src_source=src_source, source_model=source_model) }}
+    ```
 
 To create an XTS model, we will simply copy and paste the above template into a model named after the XTS we are creating. dbtvault will generate the XTS using parameters provided in the next steps.
 
-`dbt_project.yml`
-```yaml
-models:
-  my_dbtvault_project:
-   xts:
-    materialized: incremental
-    tags:
-      - xts
-    xts_customer:
-      vars:
-        ...
-```
-!!! tip "New in dbtvault v0.7.0"
-    You may also use the [vault_insert_by_period](../macros.md#vault_insert_by_period) materialisation, a custom materialisation 
-    included with dbtvault which enables you to iteratively load a table using a configurable period of time (e.g. by day). 
+We recommend setting the `incremental` materialization on all of your satellites using the `dbt_project.yml` file:
+
+=== "dbt_project.yml"
+
+    ```yaml
+    models:
+      my_dbtvault_project:
+       xts:
+        materialized: incremental
+        tags:
+          - xts
+        xts_customer:
+          vars:
+            ...
+    ```
+
+[comment]: <> (!!! tip "New in dbtvault v0.7.0")
```

Comment:
Remove these comments please.

Reply:
Removed the comments. 

The reason I commented them out in the first place, was that I was not sure whether having a Tip starting with a "New in dbtvault **0.7.0**" title is still appropriate. Let me know if you want to change/remove the Tip's title.